index.html

```diff
@@ -0,0 +1,1710 @@
+                  <h2 class="card-heading mb-2"><a href="#" class="card-link-slide">14 Ways to Save Your Financial Life</a></h2>
+                </div>
+            </div>
+
+            <div class="card col-12 col-md-4 rounded-0 border-0 mt-sm-3 pb-sm-2" >
+                <!-- <div class="slideshow-editor position-absolute">
+                  <a class="d-inline-block
+                font-weight-bold text-uppercase px-3 py-2" href="#">26</a>
+                </div> -->
+                <img class="card-img-top  position-relative rounded-0 " src="https://d.newsweek.com/en/full/1427783/dog-canoe-imagecountry-getty-images.jpg?w=767&h=512&f=fdb8a166d48091fef0408b9614b288e2" alt="Card image cap">
+                <div class="card-body">
+                  <h2 class="card-heading mb-2"><a href="#" class="card-link-slide">The Most Dog-Friendly Cities in America</a></h2>
+                </div>
+            </div>
+
+            <div class="card col-12 col-md-4 rounded-0 border-0 mt-sm-3 pb-sm-2" >
+                <!-- <div class="slideshow-editor position-absolute">
+                  <a class="d-inline-block
+                font-weight-bold text-uppercase px-3 py-2" href="#">17</a>
+                </div> -->
+                <img class="card-img-top  position-relative rounded-0 " src="https://d.newsweek.com/en/full/1391285/4-d-day.jpg?w=767&h=512&f=5665f5afc5b3684a4e26437a4accf262" alt="Card image cap">
+                <div class="card-body">
+                  <h2 class="card-heading mb-2"><a href="#" class="card-link-slide">D-Day: Photos from the Invasion of Normandy</a></h2>
+                </div>
+            </div>
+        </div>
+      </div>
+    </div>
+
+     <!-- BOTTOM STORIES -->
+    <div class="section-pick-7-times ">
+
+      <!-- US -->
+      <div class="block-title pt-4 border-top">
+        <a href="#" class="block-link">u.s.</a>
+      </div>
+      <!-- <div class="row"> -->
+        <div class="row order-0">
+          <div class=" col-12 col-md-4 content-1 order-0">
+              <div class="card rounded-0 border-0 mt-sm-2 " >
+                  <img class="card-img-top rounded-0 " src="https://d.newsweek.com/en/full/1502920/pete-buttigieg-foreign-policy-speech.jpg?w=767&h=512&f=26dcdfe5cd996001f2901c8314a7e648" alt="pete-buttigieg-foreign-policy-speech">
+              </div>
+              <h2 class="card-heading my-sm-2">
+                <a href="#" class="card-link-slide">Pete Buttigieg Says He'd Never Exchange 'Love Letters' With a Dictator</a>
+              </h2>
+          </div>
+
+          <div class=" col-12 col-md-4 content-2 order-1">
+              <div class="card rounded-0 border-0 mt-sm-2 " >
+                  <img class="card-img-top rounded-0 "
+                  src="https://d.newsweek.com/en/full/1502927/congress-pay-increase-causing-democratic-rift.jpg?w=767&h=512&f=99953b7f5bd2228aa5dd7f6c9b40be50c"
+                  alt="congress-pay-increase-causing-democratic-rift">
+              </div>
```

Comment:
It is of a good practice to make use of `footer tag` on this line, checkout documentation for that [here](https://www.themelocation.com/best-html5-practices/)

Reply:
Corrections made except for your last comment. I do not think it is necessary since your observation is what I got from bootstrap 4 card image utility.